frontend/src/map.jsx

```diff
@@ -11,12 +10,40 @@ import WKT from 'ol/format/WKT';
 import Feature from 'ol/Feature';
 import { Polygon } from 'ol/geom';
 import {
-  Select,
   Translate,
   defaults as defaultInteractions,
 } from 'ol/interaction';
+import { defaults } from 'ol/control';
+import { Fill, Stroke, Style } from 'ol/style';
+
+import { Button, Icon } from '@blueprintjs/core';
+
+import lulcLayer from './map/lulcLayer';
+import LayerPanel from './map/LayerPanel';
+import {
+  satelliteLayer,
+  streetMapLayer,
+  labelLayer,
+} from './map/baseLayers';
+import {
+  selectedFeatureStyle,
+  patternSamplerBoxStyle
+} from './map/styles';
 
-import {selectedFeatureStyle, patternSamplerBoxStyle} from './styles';
+const styleParcel = (zoom) => {
+  const style = new Style({
```

Comment:
I'd suggest defining all the styles in `map/styles.js`

Reply:
Done!